src/main/java/IP/duke/main/DukeException.java

```diff
@@ -0,0 +1,47 @@
+package IP.duke.main;
+
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+
+
+/**
+ * Represents exceptions raised in Duke.Duke.
+ * 
+ * @author Gordon Yit
+ * @version CS2103T, Semester 2.
+ */
+public class DukeException extends Exception{
```

Comment:
Should there be a space between `Exception` and `{`?
```suggestion
public class DukeException extends Exception {
```